### PR TITLE
move experiment genie_xsec dependency to sbncode

### LINKED ISSUE #15: Move dependency on GENIE cross section package from experiment repositories to sbncode
Both experiments depend on GENIE cross section, and there is value in having them share the same version of those.
Technically, the dependency can be moved into `sbncode`.

### PR DESCRIPTION
**Pull request by miquelnebot**
This is a request to move experiment genie_xsec dependency to sbncode.
Open issue details:
https://github.com/SBNSoftware/sbncode/issues/15#issuecomment-889241974
Closes #15 
ExperimentPRs:
https://github.com/SBNSoftware/sbndcode/pull/131
https://github.com/SBNSoftware/icaruscode/pull/222

### CODE REVIEW

**Review by marcodeltutto — APPROVED** [review]
Thanks Miquel!

File: ups/product_deps [hunk 1]
```diff
@@ -38,6 +38,7 @@ larsoft             v09_27_00
 sbnobj              v09_11_15_01
 sbnanaobj           v09_16_12
 sbndaq_artdaq_core  v0_07_08_of0
+genie_xsec          v3_00_04a
```
Comment (mroda88) [inline ups/product_deps:41]: While we are at it, we should update the dependency to the latest version. v3_00_06

**Review by mroda88 — CHANGES_REQUESTED** [review]
Can we give it a try with v3_00_06? The spline's dependencies are ok.

**Review by mroda88 — CHANGES_REQUESTED** [review]
I'm sorry my previous message was a little too early. There are not splines v3_00_06. The original commit was ok

**Review by mroda88 — APPROVED** [review]
Perfect!

**Review by wesketchum — APPROVED** [review]
Looks good to me, though obviously need to make sure CI tests are ok.

**Comment by jzennamo** [comment]
@miquelnebot @marcodeltutto @mroda88 Just wanting to check on this, are these 2 CI failures expected? It looks like there are a slightly different number of hits and the code is a bit faster.

**Pull request merged**